R/googledrive-package.R

```diff
@@ -42,6 +42,9 @@
 #' @importFrom lifecycle deprecated
 #' @importFrom rlang :=
 #' @importFrom rlang %||%
+#' @importFrom vctrs vec_cast
+#' @importFrom vctrs vec_ptype2
+#' @importFrom vctrs vec_restore
```

Comment:
I don't (can't) import `dplyr::dplyr_reconstruct()`, since dplyr is not in Imports. In your examples I studied, you import the entire vctrs namespace, so this is also a bit different.

Reply:
I'm very liberal with always `@import`-ing both vctrs and rlang in anything I work on, but this is completely fine